packages/expo-notifications/src/warnOfExpoGoPushUsage.ts

```diff
@@ -6,7 +6,7 @@ export const warnOfExpoGoPushUsage = () => {
   if (__DEV__ && isRunningInExpoGo() && !didWarn) {
     didWarn = true;
     console.warn(
-      `expo-notifications: Push notifications (remote notifications) functionality provided by expo-notifications will be removed from Expo Go in SDK 53. Instead, use a development build. Read more at https://docs.expo.dev/develop/development-builds/introduction/.`
```

Comment:
Text looks good! One more suggestion: have separate `warnOfExpoGoPushUsage.ios.ts` and `warnOfExpoGoPushUsage.android.ts`, and only show the warning on Android.

Reply:
I think if I developed on Expo Go for iOS, I'd like to know about android limitations sooner rather than later, so the warning if a bit more "eager" and I think it's okay 🙂

Reply:
OK sounds good